lib/va_profile/models/person.rb

```diff
@@ -20,6 +20,18 @@ class Person < Base
       attribute :updated_at, Common::ISO8601Time
       attribute :vet360_id, String
 
+      CONTACT_INFO_CHANGE_TEMPLATE = Settings.vanotify.services.va_gov.template_id.contact_info_change
```

Comment:
Can we put a new line between CONTACT_INFO_CHANGE_TEMPLATE and EMAIL_PERSONALISATIONS. It seems they run into each other

Reply:
Done!